libretroshare/src/util/rsdir.cc

```diff
@@ -516,6 +528,23 @@ bool	RsDirUtil::checkCreateDirectory(const std::string& dir)
 	return true;
 }
 
+#if __cplusplus < 201703L
+bool std::filesystem::create_directories(const std::string& path)
+{
+	for( std::string::size_type lastIndex = 0; lastIndex < std::string::npos;
```

Comment:
AFAIU `RsDirUtil::checkCreateDirectory` operate correctly on absolute paths so I expect that when it is passed "C:/" it does nothing because it already exists like it does when it is passed "/" on Linux, anyway it would be interesting to have it tested on Windows but I haven't a machine with that, but sure if doesn't work on windows we will get plenty of rants once merged :p . About the case of "some/path/" if it is passed like that I do expect that "$PWD/some/" is created if not already existing and then the same with "$PWD/some/path/"

Reply:
this function is basically the equivalent of `mkdir -p` on your shell

Reply:
you're probably right because "/" is forbidden in directory names.